.gitpod.yml

```diff
@@ -0,0 +1,27 @@
+image:
+  file: ./tools/docker_dev/.gitpod.Dockerfile
+tasks:
+  - init: |
+      # when the workspace is recreated:
+      # Scipy is built as are the docs
+      conda activate scipydev
+      python setup.py build_ext --inplace
```

Comment:
adding `-j2` or `j-4` here should help, to get parallel builds. It looks like you have multiple vCPUs available; there's some config for how many cores and how much memory you get: https://www.gitpod.io/docs/self-hosted/latest/install/workspaces/#sizing.

Reply:
That won't do anything because there's a race condition in NumPy distutils that caused Pauli to turn off the `-j` switch [in our `setup.py`](https://github.com/scipy/scipy/blob/master/setup.py#L252). You could try `NPY_NUM_BUILD_JOBS=2 python ..`, but that is a different kind of parallelism and often does not help per https://github.com/numpy/numpy/issues/15957#issuecomment-761248195

Reply:
I'm semi-sure that that disabling doesn't work; that's for example how we got ARM64 CI to not time out: https://github.com/numpy/numpy.org/issues/313#issuecomment-751466980

Also, it definitely parallelizes Cython codegen.

I'll check on the rest later.

Reply:
```
$ git clean -xdf
$ time python setup.py build_ext -i -j4
real    6m55.036s
user    12m14.623s
sys     0m50.423s

$ git clean -xdf
$ time python setup.py build_ext -i 
real    6m54.339s
user    12m15.008s
sys     0m49.768s

$ git clean -xdf
$ export NPY_NUM_BUILD_JOBS=4
$ time python setup.py build_ext -i
real    6m54.327s
user    12m14.272s
sys     0m49.136s
```

Hmm, need to fix that ....